src/style.css

```diff
@@ -1 +1,108 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
```

Comment:
don't apply anything except box-sizing to *

Reply:
How I can reset browser default margins?